src/codegen/struct_layout.rs

```diff
@@ -365,7 +365,7 @@ impl<'a> StructLayoutTracker<'a> {
             return true;
         }
 
-        if self.max_field_align >= layout.align {
+        if !self.is_packed && self.max_field_align >= layout.align {
```

Comment:
I want to check if it has explicit align in C code to avoid packed-only also get align attribute in rust side.

Reply:
I'm confused about this change, can you elaborate? What case does this make a difference on?

Reply:
Because packed or aligned cases are already being handled well, I need to check packed *and* aligned case here:
https://github.com/rust-lang/rust-bindgen/pull/2248/files#diff-01720a19d4518b9c16feea9cefbee210a853a1c99d1e9b19aec23e72877e52f0R2019-R2023

Without change, any packed type will not be marked as explicit_align. So the new handling condition doesn't hit.
The new problem introduced by this is only packed struct is also marked as aligned. Does bindgen have a feature to dinstinguish it?
Because the estimated size for clang is different, it looks like to be distinguishable.